src/functions/products/getProductsList/handler.ts

```diff
@@ -0,0 +1,12 @@
+import { formatJSONResponse } from '@libs/api-gateway';
+import { middyfy } from '@libs/lambda';
+import { APIGatewayProxyResult } from 'aws-lambda';
+import { products } from 'src/mocks/data';
+
+const getProductsList = async (): Promise<APIGatewayProxyResult> => {
+  return formatJSONResponse({
+    data: products,
```

Comment:
As per _task 3.1, 3. The response from the lambda should be a full array of products._ -  In your case its returning an object with 'data' as key and value as full array. I believe you can change this to return the correct type as specified in the task.

Reply:
It simply says "The response from the lambda should be a full array of products" so I believe anyone can interpret this in his own way.